ansible/inventory/test

```diff
@@ -1,44 +1,46 @@
 # NOTE: This file is generated by terraform.py
 # For emergency use when Consul fails
 [all]
-devel-large-01.aws-eu-central-1a.nimbus.pyrmont hostname=devel-large-01.aws-eu-central-1a.nimbus.pyrmont ansible_host=3.125.107.176 env=nimbus stage=pyrmont data_center=aws-eu-central-1a region=eu-central-1a dns_entry=devel-large-01.aws-eu-central-1a.nimbus.pyrmont.statusim.net
-devel-large-02.aws-eu-central-1a.nimbus.pyrmont hostname=devel-large-02.aws-eu-central-1a.nimbus.pyrmont ansible_host=35.156.239.243 env=nimbus stage=pyrmont data_center=aws-eu-central-1a region=eu-central-1a dns_entry=devel-large-02.aws-eu-central-1a.nimbus.pyrmont.statusim.net
-devel-small-01.aws-eu-central-1a.nimbus.pyrmont hostname=devel-small-01.aws-eu-central-1a.nimbus.pyrmont ansible_host=18.192.164.99 env=nimbus stage=pyrmont data_center=aws-eu-central-1a region=eu-central-1a dns_entry=devel-small-01.aws-eu-central-1a.nimbus.pyrmont.statusim.net
-devel-small-02.aws-eu-central-1a.nimbus.pyrmont hostname=devel-small-02.aws-eu-central-1a.nimbus.pyrmont ansible_host=18.198.163.2 env=nimbus stage=pyrmont data_center=aws-eu-central-1a region=eu-central-1a dns_entry=devel-small-02.aws-eu-central-1a.nimbus.pyrmont.statusim.net
-devel-small-03.aws-eu-central-1a.nimbus.pyrmont hostname=devel-small-03.aws-eu-central-1a.nimbus.pyrmont ansible_host=18.198.65.195 env=nimbus stage=pyrmont data_center=aws-eu-central-1a region=eu-central-1a dns_entry=devel-small-03.aws-eu-central-1a.nimbus.pyrmont.statusim.net
-devel-small-04.aws-eu-central-1a.nimbus.pyrmont hostname=devel-small-04.aws-eu-central-1a.nimbus.pyrmont ansible_host=52.58.71.58 env=nimbus stage=pyrmont data_center=aws-eu-central-1a region=eu-central-1a dns_entry=devel-small-04.aws-eu-central-1a.nimbus.pyrmont.statusim.net
+stable-large-01.aws-eu-central-1a.nimbus.pyrmont hostname=stable-large-01.aws-eu-central-1a.nimbus.pyrmont ansible_host=3.125.107.176 env=nimbus stage=pyrmont data_center=aws-eu-central-1a region=eu-central-1a dns_entry=stable-large-01.aws-eu-central-1a.nimbus.pyrmont.statusim.net
```

Comment:
No, you can't change those files by hand. To rename the hosts and change their groups Terraform configuration has to be changed, which you don't have the keys to do.

Reply:
Well, let's see if the generator will disagree with the edits I made :P